pyteal/ast/subroutine.py

```diff
@@ -1025,6 +1036,14 @@ def __call__(self, subroutine: SubroutineDefinition) -> SubroutineDeclaration:
             if not self.use_frame_pt:
                 deferred_expr = output_carrying_abi._stored_value.load()
 
+        if self.use_frame_pt:
+            assert proto.mem_layout
+            depth = len(proto.mem_layout.local_stack_types)
+            # only when we have 1 return, and with other local variables
+            # we use bury to bury the result to 0 index against frame pointer
+            if not abi_output_kwargs and 0 < proto.num_returns < depth:
+                deferred_expr = Bury(depth)
```

Comment:
Could `FrameBury` be used here instead by using the offset from the frame pointer? I ask because it appears that this is the only usage of `Bury` so if the answer is "yes", it's an argument against introducing `class Bury` and sticking to `FrameBury` for frame pointer related work.